components/tidb_query_expr/src/impl_time.rs

```diff
@@ -218,6 +224,78 @@ pub fn make_date(ctx: &mut EvalContext, year: &Int, day: &Int) -> Result<Option<
     Ok(Some(ret))
 }
 
+#[rpn_fn(capture = [extra,args])]
+#[inline]
+pub fn make_time(
+    extra: &RpnFnCallExtra,
+    args: &[crate::RpnStackNode<'_>],
+    hour: &Int,
+    minute: &Int,
+    second: &Real,
```

Comment:
`Real` is alias to `NotNan`. It cannot be nan.

Reply:
> `Real` is alias to `NotNan`. It cannot be nan.

I see,I call it because vscode notify me it has this function😂